python_modules/libraries/dagster-dbt/setup.py

```diff
@@ -37,7 +37,8 @@ def get_version() -> str:
     install_requires=[
         f"dagster{pin}",
         # Follow the version support constraints for dbt Core: https://docs.getdbt.com/docs/dbt-versions/core
-        "dbt-core>=1.6,<1.8",
+        # TODO: remove dbt 1.5.x as a lower bound once the pyright environment is less constrained.
+        "dbt-core>=1.5,<1.8",
```

Comment:
is there not a `pyright` specific way to resolve this? having the actual deps be inaccurate seems not ideal

Reply:
what happens if we remove this https://github.com/dagster-io/dagster/blob/master/pyright/master/requirements.txt#L103-L107 ?

Reply:
and https://github.com/dagster-io/dagster/blob/master/python_modules/automation/tox.ini#L64-L68

Reply:
Yeah just saw that. Removed and rebuilt the pins.